lib/plaid.js

```diff
@@ -3,7 +3,12 @@
 var environments = require('./plaidEnvironments.js');
 var Client = require('./PlaidClient.js');
 
+var isPlaidError = function(err) {
+  return err != null && err.error_code != null;
```

Comment:
👍 nice

should we check the other fields as well? as noted in the docs, all 4 fields will be present (and not null) for all plaid errors. In the interest of defining a user-defined-type-guard for our typescript clients, it is probably best to make it as strict as possible, so that its less likely for garbage that is passed to the function to product "false positives".

https://plaid.com/docs/api/#errors-overview

Reply:
makes sense — I was following README.md, which wasn't strict enough.